src/apify/proxy_configuration.py

```diff
@@ -325,7 +325,7 @@ async def _check_access(self: ProxyConfiguration) -> None:
         proxy_status_url = f'{self._actor_config.proxy_status_url}/?format=json'
 
         status = None
-        async with httpx.AsyncClient(proxies=await self.new_url()) as client:
+        async with httpx.AsyncClient(proxies=await self.new_url(), timeout=10) as client:
```

Comment:
Is this intentional? Can we parametrize somehow? Or is this a good enough default?

Reply:
Ah alright, I see that the JS version also doesn't allow to parametrize this - but uses 4 seconds.

https://github.com/apify/apify-sdk-js/blob/e49c868aaf6e1642ba40e9b10d2a532dc21a8a84/packages/apify/src/proxy_configuration.ts#L19

No biggie though, if this works here, I'm more than fine with it.

Reply:
Yeah, I think it's not necessary to parameterize it, good default should be enough.